index.js

```diff
@@ -132,53 +169,58 @@ function nowServing(line) {
   if (!line.length) {
     return "There is nobody waiting to be served!"
   }
-  return `Currently serving ${line.shift()}.`
+   return `Currently serving ${line.shift()}.`
 }
 
 const line = []
-
+let ticketNumber = 1
 // ***** Scope & Closures - Question 1 *****
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
+function takeATicketNumber(line) {
+  line.push(ticketNumber)
+  ticketNumber += 1
+  return `Welcome. You are ticket number ${ticketNumber - 1}.`
+}
+console.log(takeATicketNumber(line))
```

Comment:
Nice job!  How could you make this work if you started ticketNumber at 0?

Swap lines 184 and 185. Then remove the -1 from like 186